indico/modules/events/sessions/models/blocks.py

```diff
@@ -96,6 +96,9 @@ def can_manage(self, user, allow_admin=True):
         else:
             return False
 
+    def can_access(self, user, allow_admin=True):
+        return self.session.can_access(user, allow_admin)
```

Comment:
`allow_admin` should be a kwarg (too bad python 2 doesn't have keyword-only arguments)